docker/postgres.dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:9.6.17-alpine
+
+COPY docker/postgres/enable_btree_gist.sh /docker-entrypoint-initdb.d/
```

Comment:
I'm not sure this is necessary. I think we can include the script in a volume without having to create a separate dockerfile. You can include multiple scripts to run, for example, under `volumes` in the docker-compose file:

```
 - ./docker/postgres/enable_btree_gist.sh:/docker-entrypoint-initdb.d/enable_btree_gist.sh
 - ./docker/postgres/<any other script>:/docker-entrypoint-initdb.d/<other-script-name>.[sh|sql]
```

Reply:
@johnsonw I tried that, it requires including the script to the RPM to be installed to `/etc/iml-docker`.
This style of configuration is in line of our configuration of influxdb container.

Reply:
I'm not sure I fully understand the issue with doing it this way. If we can, I think it is best to minimize the number of docker images that we need to maintain, so if we can utilize the `docker-entrypoint-initdb.d` directory then that's probably the right approach instead of creating a new dockerfile with an image that needs to be maintained. I think all we need to do is include the `enable_btree_gist.sh` file in the iml-docker rpm. You can then specify the volume like this:
```
-  /etc/iml-docker/enable_btree_gist.sh:/docker-entrypoint-initdb.d/enable_btree_gist.sh
```

I think this would be a cleaner solution but if anyone disagrees i'm open to discuss.